nbconvert/preprocessors/execute.py

```diff
@@ -45,7 +45,7 @@ class ExecutePreprocessor(Preprocessor):
             If a cell execution takes longer, an exception (TimeoutError
             on python 3+, RuntimeError on python 2) is raised.
 
-            `None`, or `0` will disable the timeout.
+            `None` or `-1` will disable the timeout.
```

Comment:
Do we have single-backticks-are-double-backticks configured for nbconvert's docs? If not, I'd be fine with just leaving these un-decorated. Or configure that option.